backend/tests/v5/unit/processors/teamspaces/projects/models/federations.test.js

```diff
@@ -46,7 +46,7 @@ const federationSettings = {
 			code: 'FED1',
 		},
 		status: 'ok',
-		subModels: [{ model: 'model1' }, { model: 'model2' }],
+		subModels: [{ model: 'container1' }, { model: 'container2' }],
```

Comment:
this doesn't look like the correct schema?
models should be just an array of strings, not array of objects